src/website/app/demos/utils/examples/createThemedComponent.js

```diff
@@ -16,33 +16,24 @@
 
 /* @flow */
 import React from 'react';
-import Link from '../../../../../Link';
 import { createThemedComponent } from '../../../../../utils';
 import Sample from '../components/Sample';
-import Prose from '../components/Prose';
 import Code from '../components/Code';
 
 export default {
   id: 'create-themed-component',
   title: <Code>createThemedComponent(component, theme)</Code>,
```

Comment:
The `Code` component needs a `word-break: break-all;` in order to not break layout on small screens.